### ISSUE TITLE
New unrecoverable error?

### ISSUE THREAD

**Issue by JonScarlet** [narc0tiq/YARM#158]
I was switching from playing with Angels Mods to just the regular game without Angels, but when I loaded up my save for that, this is the error I got.

https://gyazo.com/d1e97e91f33e7ef5f8f9fe5bd160925e

Aside from updating the mod, I only disabled the Angels mods I have. Any ideas what might've caused this?

EDIT: Also not sure how to upload my save if/when you need that, though...

**Comment by wchristian** [comment]
that's a very specific combination of update on an old game. i'll release a fix in a few minutes

**Comment by wchristian** [comment]
thanks for the report

fixed version is up :)

**Comment by JonScarlet** [comment]
Alright, so that fix helped me get to starting it up. However, when I was trying to re-do some of the monitor (I loaded said base mod without YARM at the time, and I just got back to my computer), I got a new error.

[https://gyazo.com/1a37f496bd65f2eb1db2a877e458ba6c](url)

The tags I had from the previous use of YARM were still there, but the actual monitoring was gone; I had no stats in the top-left corner of my screen at all despite having over a dozen sites on this save. So as I was attempting to re-do these site, this error came up at about the 5th or 6th one.

So far this hasn't popped up again on a second attempt, so I guess I must've just accidentally tried to YARM the wrong thing or something.
